apiron/endpoint/endpoint.py

```diff
@@ -154,3 +152,6 @@ def get_merged_params(self, supplied_params=None):
 
     def __str__(self):
         return self.path
+
+    def __repr__(self):
+        return "{klass}(path='{path}')".format(klass=self.__class__.__name__, path=self.path)
```

Comment:
Why use `.format` instead of f-string?

Reply:
I assume to support older Python 3 versions, but it's a good time to specify the Python versions `apiron` will support.

Reply:
Yep, `apiron` supports 3.5+ currently and there isn't a compelling reason to stop doing that.